lib/ic/interface/lan8720a.yaml

```diff
@@ -0,0 +1,32 @@
+datasheet: http://ww1.microchip.com/downloads/en/DeviceDoc/8720a.pdf
+description: RMII 10/100 Ethernet Transceiver 
+footprint: agg:QFN-24-EP-MICROCHIP
+name: LAN8720A
+ordercodes:
+- [Farnell, '2292577']
+pins:
+- - - [VDD1A, 19, pwrin]
+    - [VDD2A, 1, pwrin]
+    - [VDDCR, 6, pwrin]
```

Comment:
This pin is supplied internally most of the time (unless you have the regulator off for whatever reason) so might be better as a passive pin.